knowledge-bases/5_customized-rag-retreive-api-langchain-claude-v2-evaluation-ragas.ipynb

```diff
@@ -45,9 +45,9 @@
     "workﬂows on top of the semantic search results. The output of the `Retrieve API` includes the the `retrieved text chunks`, the `location type` and `URI` of the source data, as well as the relevance `scores` of the retrievals. \n",
     "\n",
     "\n",
-    "We will then use the text chunks being generated and augment it with the original prompt and pass it through the `claude-instant-v1` model using prompt engineering patterns based for your use case.\n",
+    "We will then use the text chunks being generated and augment it with the original prompt and pass it through the `claude-v2.1` model using prompt engineering patterns based for your use case.\n",
```

Comment:
We can use claude-instant-v1 to generate answers and claude-v2.1 for evaluating the answers.